infra/terraform/modules/data_access/lambda_organization_events.tf

```diff
@@ -0,0 +1,84 @@
+# Zip the lambda function before the actual deploy
+data "archive_file" "organization_events_zip" {
+    type        = "zip"
+    source_dir  = "${path.module}/organization_events"
+    output_path = "/tmp/organization_events.zip"
+}
+
+# Lambda function which handles GH organization events
+resource "aws_lambda_function" "organization_events" {
+    description = "Updates IAM resources in response to GH organization events"
+    filename = "/tmp/organization_events.zip"
+    source_code_hash = "${data.archive_file.organization_events_zip.output_base64sha256}"
+    function_name = "${var.env}_organization_events"
+    role = "${aws_iam_role.organization_events_role.arn}"
+    handler = "organization_events.event_received"
+    runtime = "python3.6"
+    timeout = 10
+    depends_on = ["data.archive_file.organization_events_zip"]
+    environment {
+        variables = {
+            CREATE_ROLE_ARN = "${aws_lambda_function.create_user_role.arn}",
+            DELETE_ROLE_ARN = "${aws_lambda_function.delete_user_role.arn}",
```

Comment:
I kept these names sweet and short but I get these may be confused for "roles", not "lambdas"'s  ARNs so I'll change.